eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,11 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath "DevDiv\ASP.NET Core"
+-TsaIterationPath DevDiv
+-TsaRepositoryName AspNetCore
+-TsaCodebaseName AspNetCore
```

Comment:
Does it matter that these names don't match the case of the repos? And, since we run SDL only internally, shouldn't the names be `dotnet-aspnetcore` and similar for the other repos?

Reply:
I just matched what was in the variable group, I believe they're just a convenience for identification

Reply:
It does not need to match the repo name, this is for TSA to identify the code base name to tag. This is the name you would see in the TSA emails that are sent out as notification.